client/components/payment/nextPaymentDetails.tsx

```diff
@@ -34,13 +35,14 @@ export const getNextPaymentDetails = (
 			? paidFuturePlans[1]
 			: paidFuturePlans[0];
 	if (isPaidSubscriptionPlan(mainPlan)) {
-		const paymentKey = `Next ${augmentInterval(
+		const paymentInterval =
 			subscription.currentPlans.length !== 0 &&
-				isSixForSix(mainPlan.name) &&
-				planAfterMainPlan
+			isSixForSix(mainPlan.name) &&
```

Comment:
Could we maybe split this up? I find it hard to understand what the logic is here

Reply:
Yeah, I completely agree that the logic in here is hard to follow. I think any refactoring of this is probably outside the scope of this PR though. All I've done here is to move the existing logic into a separate variable so we can get the interval (ie. `month`) in isolation from the existing `paymentKey` (ie.`Next monthly payment`)

A lot of the interval logic is tied into whether it's a six for six plan, but I need to find out whether that is still a thing. I don't believe it's something you can sign up for anymore, but there may be readers still on that plan that we need to accommodate.